internal/aez/heater_regulators.go

```diff
@@ -23,18 +23,6 @@ type HTR struct {
 	HTR2A  htr
 	HTR2B  htr
 	HTR2OD htr
-	HTR3A  htr
```

Comment:
wow :cry:  that would have been very nice not to have put in all that from the beginning.